Makefile

```diff
@@ -157,12 +157,15 @@ opa-fmt: ## Lint the OPA policies
 	$(CMD_PREFIX) docker run --platform linux/x86_64 --rm -v $(CURDIR):/workdir -w /workdir docker.io/openpolicyagent/opa:latest fmt --write $(policies)
 
 .PHONY: generate
-generate: gen-docs ## Run all code generators and formatters
+generate: .generate ## Run all code generators and formatters
+
+.generate: .gen-docs
```

Comment:
don't you need the source file added as deps here so that it does kick off if you have any source modifications?

Reply:
The source files are dependencies of `.gen-docs`. So, changed source causes `.gen-docs` to get rebuilt, which causes `.generate` to get rebui.d